src/Polly/AsyncPolicy.TResult.cs

```diff
@@ -1,4 +1,5 @@
-﻿namespace Polly;
+﻿#nullable enable
```

Comment:
Is the plan to remove these and just set it in the project file once it's "done"?

Reply:
yep. `#nullable enable` allows us to gradually transition